portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="setBackgroundColor()">
```

Comment:
The less JS parsing in HTML the more organized it is between the separation of HTML/CSS and JS. 
https://stackoverflow.com/questions/191157/window-onload-vs-body-onload